### PR TITLE
Add separate function to retrieve original version and corresponding tests

### PR DESCRIPTION
**Pull request by iverberk**

### CODE REVIEW

**Comment by iverberk** [comment]
This adds functionality described in https://github.com/naneau/semver/issues/12#issuecomment-72031262

File: Tests/VersionTest.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * @category        Naneau
+ * @package         SemVer
+ */
+
+use Naneau\SemVer\Parser;
+
+use \PHPUnit_Framework_TestCase as TestCase;
+
+/**
+ * SortTest
```
Comment (naneau) [inline Tests/VersionTest.php]: Docblock is incorrect (class name is wrong), same goes for `SortTest.php` above.

File: Tests/VersionTest.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * Testing SemVer sorting
+ *
+ * @category        Naneau
+ * @package         SemVer
+ */
+class VersionTest extends TestCase
```
Comment (naneau) [inline Tests/VersionTest.php]: Perhaps a better name is `OriginalVersionTest`

**Comment by naneau** [comment]
Two small comments, but otherwise seems to fix the regression.

**Comment by naneau** [comment]
I took the liberty of applying some fixes myself, see #17. Also, I noticed your editor is set to use tabs, which I have replaced with spaces, as per PSR.

**Pull request closed without merging**